### ISSUE TITLE
Multiple shabads short link not works

### ISSUE THREAD

**Issue by navdeepsingh** [KhalisFoundation/sttm-web#1437]
**Describe the bug**
Short link for multiple shabads not working now, it redirect to homepage instead.

**To Reproduce**
1. Go to https://www.sikhitothemax.org/shabad?id=501,167&highlight=7013,2506
2. Click on Copy link (see image below)
![Screen Shot 2021-05-17 at 10 50 04 AM](https://user-images.githubusercontent.com/2968787/118436089-c4394200-b6fd-11eb-920b-c76c33205b12.png)
3. Open this link in new tab
4. See redirection to homepage

**Expected behavior**
Like this http://sttm.co/s/5478/ redirects to https://www.sikhitothemax.org/shabad?id=5478, same behaviour expected for multiple shabads.

**Comment by tsingh777** [comment]
Should be fixed in https://github.com/KhalisFoundation/sttm-shortener

**Comment by navdeepsingh** [comment]
> Should be fixed in https://github.com/KhalisFoundation/sttm-shortener

Yes, sent a PR here https://github.com/KhalisFoundation/sttm-shortener/pull/9, please review it.

**Comment by saintsoldierx** [comment]
This is not working in production @gurjit03
